From 1ab938e762ee6ebd3211c22ef25a7575e37791c4 Mon Sep 17 00:00:00 2001
From: Baruch Siach <baruch@tkos.co.il>
Date: Thu, 29 Dec 2016 20:16:53 +0200
Subject: [PATCH] androiddump: add missing sys/time.h header

struct timeval requires sys/time.h. Fixes the following musl libc build error:

androiddump.c: In function 'useSndTimeout':
androiddump.c:262:18: error: variable 'socket_timeout' has initializer but incomplete type
     const struct timeval socket_timeout = {
                  ^

Change-Id: I52e204f32a012aabea2e54877e564576c072fe08
Signed-off-by: Baruch Siach <baruch@tkos.co.il>
---
Upstream status: https://code.wireshark.org/review/19460
---
 extcap/androiddump.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/extcap/androiddump.c b/extcap/androiddump.c
index 8287c4bb1a76..fab7c10e05cc 100644
--- a/extcap/androiddump.c
+++ b/extcap/androiddump.c
@@ -30,6 +30,7 @@
 #include <string.h>
 #include <errno.h>
 #include <time.h>
+#include <sys/time.h>
 
 #ifdef HAVE_NETINET_IN_H
 #    include <netinet/in.h>
-- 
2.11.0

